### PR TITLE
Update print statement with log statement.

### LINKED ISSUE #66: Update all print statement into log statement
We have implemented logging functionality using `zap` in #52. Now we need to convert all print statements using logger.

### PR DESCRIPTION
**Pull request by sudiptob2**
closes #66

### CODE REVIEW

File: cmd/event-collector/main.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func main() {
 
 	k, err := loadConfig(cfgFile)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+		log.Logger.Error(fmt.Sprintf("Error: %v", err))
```
Comment (dejanzele) [inline cmd/event-collector/main.go:71]: Use either `Errorf` or `Errorw` depending how you want to include the error message, don't use `Sprintf` to format the message

File: cmd/event-collector/main.go [hunk 2]
```diff
@@ -114,7 +114,7 @@ func main() {
 
 	repo, err := repository.NewECRepo(ctx, &ecConfig)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "could not create db repository: %v\n", err)
+		log.Logger.Error(fmt.Sprintf("could not create db repository: %v", err))
 		os.Exit(1)
```
Comment (dejanzele) [inline cmd/event-collector/main.go:118]: this should be a `panic(err)`

File: cmd/event-collector/main.go (hunk 1 above)
Comment (dejanzele) [reply cmd/event-collector/main.go:71]: Also no need to have `Error:` in the message, it will be logger as error already

Comment (dejanzele) [reply cmd/event-collector/main.go:71]: this applies to the rest also

Comment (dejanzele) [reply cmd/event-collector/main.go:71]: and use lowercase messages, I see lowercase everywhere (don't capitalise)

Comment (sudiptob2) [reply cmd/event-collector/main.go:71]: Hi, I tried the above suggestion but am having some problems. `log.Logger.Error()` receives a string whereas, `fmt.Errorf()` returns an `Error`. In that case, how can we use the logger?

**Review by dejanzele — APPROVED** [review]

**Pull request merged**